mitiq/shadows/test/test_shadows_utils.py

```diff
@@ -0,0 +1,72 @@
+import cirq
+import numpy as np
+import mitiq
+import pytest
+from mitiq.shadows.shadows_utils import (
+    n_measurements_tomography_bound,
+    n_measurements_opts_expectation_bound,
+    fidelity,
+    transform_to_cirq_paulistring,
+)
```

Comment:
Best to group the `mitiq` imports.

Reply:
sorry, I don't quite understand.  do you mean `from mitiq.shadows.shadows_utils import *`?

Reply:
Group imports from the same library into adjacent lines. Typically in the order of
1. Standard library imports
2. Third party imports
3. First party imports

So this example would look like this:
```py
import pytest

import cirq
import numpy as np

import mitiq
from mitiq.shadows.shadows_utils import (
    n_measurements_tomography_bound,
    n_measurements_opts_expectation_bound,
    fidelity,
    transform_to_cirq_paulistring,
)
```